packages/manager/modules/pci/src/projects/project/storages/databases/components/tags-input/tags-input.controller.js

```diff
@@ -0,0 +1,36 @@
+import remove from 'lodash/remove';
+
+export default class {
+  $onInit() {
+    this.items = [];
+    this.tag = '';
+    this.model.forEach((element) => {
+      this.items.push({ title: element });
+    });
+  }
+
+  addTag() {
```

Comment:
should it be possible to have tag as a function parameter ? reading the coding i don't see where tag comes from and why it is set to empty string after being added

Reply:
The function addTag uses the tag defined in the model which is updated in the input field. When clicking on add, the tag is added to a list of chips to be displayed and emptied so you can add other tags.
Let me know if the behavior of this component is still unclear.